stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,106 @@
+- the clicked/pressed letter must be disabled and it must be reflected in the UI _(for example, by using a different color on the disabled button)_;
+
+6. The body parts appear on the gallows in the following order: head, body, left arm, right arm, left leg, right leg. It means that the user has _6 attempts_ to guess the word.
+7. The player will continue guessing letters until either _the word is solved_ or _all six body parts_ are on the gallows.
+8. At the end of the game, a _modal window_ must appear. It must:
+
+- include the _message_ that either congratulates the user on winning the game or informs them that they failed to win;
+- the _secret word_ (even if it was guessed correctly);
+- a _"play again"_ button.
+
+9. When the user clicks on "play again", the game _starts over_:
+
+- the gallows is _empty_;
+- the question is _changed_;
+- the underscores are _updated_ according to the new secret word.
+- an incorrect guesses counter is _set to 0_.
+
+### Approximate appearance
+
+_This example is purely illustrative. Feel free to change the layout as you wish._
+![screenshot](hangman.png)
+
+### Helpful materials
+
+At the [following link](https://www.figma.com/file/ug2NAUiXPpaFDvch5TWUxd/Hangman-game?type=design&node-id=0%3A1&mode=design&t=4Gj7Ayo0yckppNo8-1) you will find:
+
+- examples of the game flow,
+- body parts (for those who don't want to design them by themselves).
+
+### Main functional requirements
+
+- The design must be _adaptive (or responsive)_, that includes desktop(1440px <= width), tablet(768px <= width < 1440px) and mobile(360px <= width < 768px). When switching between versions everything must be displayed correctly, all functionality must be present, no content must disappear or be left beyond the screen.
+- Initially, body in the index.html file _must be empty_, all necessary elements are generated using JS.
+- At the beginning of the game, the question-answer pair must be _chosen randomly_.
+- The application can be done _in English or in Russian_.
+
+## Repository requirements
+
+- The task must be done in private school's repository.
+- The code must be committed to a separate branch named as the task.
+- The `main` (or old style master) branch must be empty (contain only files like README.md or .gitignore).
+- Commit messages must follow the [guideline](https://docs.rs.school/#/git-convention)
+- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
+- The pull request name must contain **the task name**.
+- The pull request description must be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
+
+## Technical requirements
+
+- The application is displayed correctly and functions properly in the latest version of Google Chrome.
+- It is **not allowed** to use:
```

Comment:
By the way, should there be any penalty in case the html is not empty? Or is it only 20 points as one of the requirements?

Reply:
@stardustmeg it will be 0 points for the task (the whole idea about the task is to generate everything dynamically).
@helenakrasnova will add the penalties, thanks for reminding!

Reply:
[elukashova](https://github.com/elukashova) 
Is using html files as templates for components (for example, like in angular) a reason for a penalty?

Reply:
@bloodsuckers-spb thanks for reminding me about this "edge case", will add this part.

Reply:
done